privaterelay/models.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Invitations(models.Model):
+    email = models.EmailField(db_index=True)
+    date_added = models.DateTimeField(auto_now_add=True)
+    active = models.BooleanField(default=False)
+    date_sent = models.DateTimeField(null=True, blank=True)
+    date_redeemed = models.DateTimeField(null=True, blank=True)
+
+    def __str__(self):
+        return 'Invitation for %s' % self.email
+
+
+class MonitorSubscriber(models.Model):
+    class Meta:
+        db_table = 'subscribers'
+        managed = False
+
+    primary_email = models.CharField(max_length=255)
```

Comment:
```suggestion
    primary_email = models.EmailField()
```
Not very familiar with using synchronized data from another db but can we specify Django model level that the `primary_email` is a valid email? Also, can we add `db_index=True` to the `primary_email` field?

Reply:
I decided on `CharField` because this database and the data in it are node.js/knex tables. Since I'm not sure that the Monitor email validation rules and the Django email validation rules are the same, I decided to just go with `CharField`.